_posts/2021-02-01-patri-monitoring-hives-ble-and-android-app.md

```diff
@@ -0,0 +1,97 @@
+published: false
+status: publish
+category: Embedded systems
+tags: [TrackBees]
+author:
+  login: jpatri
+  email: jpatri@cc.isel.ipl.pt
+  display_name: João Pedro Patriarca
+  image: patri.jpg
+  role: Software Developer
+---
+
+The Bluetooth Low Energy (BLE) module integration in the equipment enhanced the functional characteristics to a richer and more interesting level than those achieved in previous versions to this integration. This integration also motivated the development of an Android App.that revolutionized the interaction with the equipment. This post presents the integration result of this module in terms of hardware and the interaction achieved with Android App.
+
+## Motivation
+
+[The experience carried out in real environment](https://cc.isel.pt/embedded%20systems/2020/12/20/patri-monitoring-hives-test-with-bees-colony/) demonstrated the relevance of being able to configure the equipment remotely, that is, to have the ability to establish a short distance and wireless connection between the equipment and the device that configures it. The device that configures the equipment corresponds to a mobile phone that is paired with the equipment at an early stage of configuration. The technology selected to establish a short distance and wireless connection was the Bluetooth technology, more specifically Bluetooth Low Energy (BLE).
+
+This technology has been selected because it is present in most smartphones, communicates by radio frequency not requiring the equipment to be in line of view, supports connections up to 10 meters and is low consumption. Other technologies, such as Near Field Connection (NFC), Infrared or WIFI, conditionate the desired operability in different aspects. NFC technology requires a proximity of less than 1 meter, infrared requires that the equipment be in line of view and WIFI imposes a higher consumption than the consumption imposed with the BLE not introducing any additional advantage (assuming a direct connection between the equipment and the smartphone).
+
+## Hardware solution
+
+The first TrackBees board to have integrated a BLE module was the V0.3 version, designed in August 2015. It used the [BLE112 module](https://www.silabs.com/wireless/bluetooth/bluegiga-low-energy-legacy-modules/device.ble112) from Silicon Labs (at the time, that module implemented the BLE4.0 specification). From version V0.8 (October 2017) to the last developed version has been used the [HM-10 module](http://jnhuamao.cn/bluetooth.asp?id=1) from Jinan Huamao Technology. The figures below present two versions, V0.4 and V0.8, each one with their respective BLE module highlighted (for illustration purposes, the microcontroller, GSM and GNSS modules are not mounted).
+
+![Trackbees v0.4 board](/assets/blog/2021-01-06-patri-monitoring-hives-post5-ble-and-android-app_TB_v0.4.png "Trackbees v0.4 board")![Trackbees v0.8 board](/assets/blog/2021-01-06-patri-monitoring-hives-post5-ble-and-android-app_TB_v0.8.png "Trackbees v0.8 board")
+
```

Comment:
Maybe place these two figures side by side. I think you can achieve it with a table with 2 columns, such as:
```html
<table>
<tr>
<td>
<img src="...." witdth="200px">
</td>
<td>
<img src="..." witdth="200px">
</td>
</tr>
</table>
```